sjsonnet/src/sjsonnet/Std.scala

```diff
@@ -324,6 +324,17 @@ object Std {
     builtin("strReplaceAll", "str", "from", "to"){ (ev, fs, str: String, from: String, to: String) =>
       str.replaceAll(from, to)
     },
+
+    builtin("rstripChars", "str", "chars"){ (ev, fs, str: String, chars: String) =>
+      str.replaceAll("[" + chars + "]+$", "")
+    },
+    builtin("lstripChars", "str", "chars"){ (ev, fs, str: String, chars: String) =>
+      str.replaceAll("^[" + chars + "]+", "")
+    },
+    builtin("stripChars", "str", "chars"){ (ev, fs, str: String, chars: String) =>
+      str.replaceAll("[" + chars + "]+$", "").replaceAll("^[" + chars + "]+", "")
```

Comment:
Would I be right to say that this won't work if `chars` contains `]`? I think an approach using `str.dropWhile(!chars.contains(_)).reverse.dropWhile(!chars.contains(_)).reverse` would avoid that problem

Reply:
Using `indexWhere(!chars.contains(_))` and `lastIndexWhere(chars.contains(_))` to shorten the string would probably work too, and much faster than `dropWhile` and `takeWhile`. We should also make sure to convert `chars` into a `Set` to ensure fast lookup

Reply:
Ah, you were absolutely right, my bad - but `Regex.quote()` takes care of all special chars :)